<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageCapturePropertiesWindow</class>
 <widget class="QDialog" name="ImageCapturePropertiesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>327</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Image Quality</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rdBtnImageQualityVLow">
        <property name="text">
         <string>Very Low</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdBtnImageQualityLow">
        <property name="text">
         <string>Low</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdBtnImageQualityNormal">
        <property name="text">
         <string>Normal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdBtnImageQualityHigh">
        <property name="text">
         <string>High</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdBtnImageQualityVHigh">
        <property name="text">
         <string>Very High</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="grpBoxFileFormat">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>File Format</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="rdBtnFileFormatJPEG">
        <property name="text">
         <string>jpeg</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdBtnFileFormatPNG">
        <property name="text">
         <string>png</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdBtnFileFormatWEBP">
        <property name="text">
         <string>webp</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdBtnFileFormatTIFF">
        <property name="text">
         <string>tiff</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Image Size - //TODO</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QGroupBox" name="grpBoxImgSize16">
        <property name="title">
         <string>16:9</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="rdBtnImgSize16_640">
           <property name="text">
            <string>640x360</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rdBtnImgSize16_1280">
           <property name="text">
            <string>1280x720</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rdBtnImgSize16_1366">
           <property name="text">
            <string>1366x768</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rdBtnImgSize16_1600">
           <property name="text">
            <string>1600x900 </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rdBtnImgSize16_1920">
           <property name="text">
            <string>1920x1080</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="grpBoxImgSize4">
        <property name="title">
         <string>4:3</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QRadioButton" name="rdBtnImgSize4_640">
           <property name="text">
            <string>640x480</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rdBtnImgSize4_1024">
           <property name="text">
            <string>1024x768</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rdBtnImgSize4_1440">
           <property name="text">
            <string>1440x1080</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rdBtnImgSize4_1600">
           <property name="text">
            <string>1600x1200</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rdBtnImgSize4_2048">
           <property name="text">
            <string>2048 x1536</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="rdBtnSelect16">
        <property name="text">
         <string>16:9</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rdBtnSelect4">
        <property name="text">
         <string>4:3</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSave">
       <property name="text">
        <string>Save preferences</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
